recipes/client_common.rb

```diff
@@ -0,0 +1,33 @@
+# Add default repos for Bareos Packages
+include_recipe 'bareos::package_repos_common'
+
+package 'bareos-filedaemon'
+
+directory '/etc/bareos/bareos-fd.d' do
+  path '/etc/bareos/bareos-fd.d'
+  owner 'root'
+  group 'bareos'
+  mode '0750'
+  action :create
+end
```

Comment:
does the filedaemon package not put this down?  or are you mucking with owners/permissions?

Reply:
I am enforcing them. That's all.